win32ss/printing/base/winspool/printers.c

```diff
@@ -424,10 +424,9 @@ EnumPrintersA(DWORD Flags, PSTR Name, DWORD Level, PBYTE pPrinterEnum, DWORD cbB
     TRACE("EnumPrintersA(%lu, %s, %lu, %p, %lu, %p, %p)\n", Flags, Name, Level, pPrinterEnum, cbBuf, pcbNeeded, pcReturned);
 
     // Check for invalid levels here for early error return. MSDN says that only 1, 2, 4, and 5 are allowable.
-    if (Level !=  1 && Level != 2 && Level != 4 && Level != 5)
+    if (Level == 0 || Level == 3 || Level > 5)
```

Comment:
Do I need to point out what's wrong here?

Reply:
> Do I need to point out what's wrong here?

Reverted, though I think consistency would be worth it here.